### PR TITLE
Fix: Floating Buttons - Product QA fixes

### PR DESCRIPTION
**Pull request by nicoladj77**
## PR Checklist

- [x] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

* Prevent beforeunload message when user closes the remote library after making changes to the doc
* When deleting the widget, also delete the container
* If user reloads an empty page, show the library
* Fix issue with the advanced tab
* Ensure Floating button is open only when editing a Floating Button document

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [x] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

File: core/common/assets/js/views/modal/header.js [hunk 1]
```diff
@@ -37,6 +37,8 @@ export default class extends Marionette.LayoutView {
 		this._parent._parent._parent.hideModal();
 
 		if ( this.isFloatingButtonLibraryClose() ) {
+			// Had to use jQuery as it is the only thing that worked.
```
Comment (louiswol94) [inline core/common/assets/js/views/modal/header.js:40]: Please remove this comment

File: assets/dev/scss/editor/preview/preview.scss [hunk 2]
```diff
@@ -18,7 +18,7 @@
 @import "inline-editor";
 
 @import "../../../../../modules/elements-color-picker/assets/scss/elements-color-picker";
-@import "../../../../../modules/floating-buttons/assets/scss/preview/floating-buttons-preview";
+@import "../../../../../modules/floating-buttons/assets/scss/preview/contact-buttons-preview";
```
Comment (louiswol94) [inline assets/dev/scss/editor/preview/preview.scss:21]: Are we sure this needs to be renamed? It looks like it's actually related to the Floating Buttons Document and not widget specific.

File: modules/floating-buttons/assets/js/frontend/handlers/contact-buttons.js [hunk 3]
```diff
@@ -342,8 +342,10 @@ export default class ContactButtonsHandler extends Base {
 			}
 		}
 
-		// Default to open in Editor
-		if ( elementorFrontend.isEditMode() ) {
+		if (
+			elementorFrontend.isEditMode() &&
+			'floating-buttons' === elementor?.config?.document?.type
```
Comment (louiswol94) [inline modules/floating-buttons/assets/js/frontend/handlers/contact-buttons.js:347]: Why do we need to check the doc type here?

**Review by louiswol94 — CHANGES_REQUESTED** [review]

File: assets/dev/scss/editor/preview/preview.scss (hunk 2 above)
Comment (nicoladj77) [reply assets/dev/scss/editor/preview/preview.scss]: @louiswol94 agreed and changed

File: modules/floating-buttons/assets/js/frontend/handlers/contact-buttons.js (hunk 3 above)
Comment (louiswol94) [reply modules/floating-buttons/assets/js/frontend/handlers/contact-buttons.js:347]: Do you mean it renders in the Editor as well? When you are editing another page?

File: core/common/assets/js/views/modal/header.js (hunk 1 above)
Comment (louiswol94) [reply core/common/assets/js/views/modal/header.js]: @nicoladj77 did it not work to autosave?

Comment (nicoladj77) [reply core/common/assets/js/views/modal/header.js]: @louiswol94 no didn't work.

File: modules/floating-buttons/assets/js/frontend/handlers/contact-buttons.js (hunk 3 above)
Comment (nicoladj77) [reply modules/floating-buttons/assets/js/frontend/handlers/contact-buttons.js:347]: the problem is that it renders as open, while normally on the FE it is not open. the only case when it hsould be open by default is when editing it as a floting button doc

File: modules/floating-buttons/module.php [hunk 4]
```diff
@@ -8,6 +8,7 @@
 use Elementor\Core\Base\Module as BaseModule;
 use Elementor\Core\Documents_Manager;
 use Elementor\Core\Experiments\Manager;
+use Elementor\Core\Experiments\Manager as Experiments_Manager;
```
Comment (louiswol94) [inline modules/floating-buttons/module.php:11]: Already imported in line 10

File: modules/floating-buttons/module.php [hunk 5]
```diff
@@ -55,11 +56,15 @@ public static function get_experimental_data(): array {
 			'name' => static::EXPERIMENT_NAME,
 			'title' => esc_html__( 'Floating Buttons', 'elementor' ),
 			'description' => esc_html__( 'Boost visitor engagement with Floating Buttons. The Floating Button template library offers a variety of interactive one-click contact options, highlighted links, and calls to action to increase your website conversions.', 'elementor' ),
-			'hidden' => true,
 			'default' => Manager::STATE_INACTIVE,
+			'release_status' => Experiments_Manager::RELEASE_STATUS_BETA,
```
Comment (louiswol94) [inline modules/floating-buttons/module.php:60]:
```suggestion
			'release_status' => Manager::RELEASE_STATUS_BETA,
```

**Review by louiswol94 — APPROVED** [review]

**Pull request merged**